src/renderers/webxr/WebXRManager.js

```diff
@@ -730,6 +739,32 @@ class WebXRManager extends EventDispatcher {
 
 				}
 
+				const depthData = glBinding.getDepthInformation( views[0] );
+				if ( depthData.isValid ) {
+
+					if (glDepthTexture === null) {
+						glDepthTexture = depthData.texture;
+
+						const depthTexture = new Texture();
+						const texProps = renderer.properties.get(depthTexture);
+						texProps.__webglTexture = glDepthTexture;
+
+						if (( depthData.depthNear != session.renderState.depthNear ) || ( depthData.depthFar != session.renderState.depthFar )) {
+
+							session.updateRenderState( {depthNear: depthData.depthNear, depthFar: ( depthData.depthFar > 10000 ? 10000 : depthData.depthFar ) } );
```

Comment:
this seems like a WebXR bug

Reply:
Spec looks like it allows infinity:

> If activeState’s [depthFar](https://immersive-web.github.io/webxr/#dom-xrrenderstate-depthfar) is greater than session’s [maximum far clip plane](https://immersive-web.github.io/webxr/#maximum-far-clip-plane) set activeState’s [depthFar](https://immersive-web.github.io/webxr/#dom-xrrenderstate-depthfar) to session’s [maximum far clip plane](https://immersive-web.github.io/webxr/#maximum-far-clip-plane).

And

> The [maximum far clip plane](https://immersive-web.github.io/webxr/#maximum-far-clip-plane) SHOULD be greater than 1000.0 (and MAY be infinite).

So if it's not working then it's a Chromium implementation issue. Idea is to allow `depthFar: Infinity`, right?

Reply:
> So if it's not working then it's a Chromium implementation issue. Idea is to allow `depthFar: Infinity`, right?

Yes. If I pass infinity, I get an error saying the value is non-finite